### PR TITLE
Feat: Add qiblat certified field

### PR DESCRIPTION
**Pull request by ismi-abbas**
- Along with this changes, there are some minor UI addition and changes

### CODE REVIEW

**Comment by farhan-helmy** [comment]
resolve conflict

File: prisma/schema.prisma [hunk 1]
```diff
@@ -21,22 +21,35 @@ model Example {
...
+    images              SurauPhoto[]
+    ratings             Rating[]
+    state_id            String
+    state               State        @relation(fields: [state_id], references: [id], onDelete: Cascade)
+    district_id         String
+    district            District     @relation(fields: [district_id], references: [id], onDelete: Cascade)
+    mall_id             String?
+    mall                Mall?        @relation(fields: [mall_id], references: [id], onDelete: Cascade)
+    created_at          DateTime     @default(now())
+    updated_at          DateTime     @default(now()) @updatedAt
```
Comment (farhan-helmy) [inline prisma/schema.prisma:41]: ni boleh remove sebab kat atas da ada, dia conflict

**Review by farhan-helmy — CHANGES_REQUESTED** [review]

File: src/components/AddSurauForm.tsx [hunk 2]
```diff
@@ -309,11 +381,96 @@ const AddSurauForm: FC<AddSurauFormProps> = ({ setOpen }) => {
...
+                            className="block w-full flex-1 rounded-md border-gray-300 focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
+                            placeholder="0.000"
+                            onChange={(e) => {
+                              setLongitude(parseFloat(e.target.value));
+                            }}
+                          />
+                        </div>
+                      </div>
+                      <div>
+                        <label className="block text-sm font-medium text-gray-700">
```
[the remark this answers is not in the document]
Comment (ismi-abbas) [reply src/components/AddSurauForm.tsx:444]: Ok, reduce gap untuk small screen.

File: prisma/schema.prisma (hunk 1 above)
Comment (ismi-abbas) [reply prisma/schema.prisma]: Removed

**Pull request merged**